content/rs/administering/designing-production/performance/shard-placement.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: Configuring Shard Placement
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Default Shard Placement Policy
+
+The cluster configuration includes a default policy for the shard placement policy.
```

Comment:
Maybe: "When you set up a Redis Enterprise cluster, you specify a default shard placement policy that will be applied to all new databases."

Reply:
It's not necessary to choose the default shard placement policy.